docker-hub/builds.md

```diff
@@ -231,4 +231,4 @@ OK
 ```
 
 To verify everything is working correctly, check the **Last 10 Trigger Logs** on
-the page.
\ No newline at end of file
+the page.
```

Comment:
I did not manually change this, the GitHub UI apparently did this change for me.